src/Components/DisplayQuotes/Displayqoutes.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import './Displayqoutes.css';
+import axios from 'axios';
+
+function Displayqoutes() {
```

Comment:
- [ ] I get this error when I run your app. Kindly inspect and fix it. And the app displays nothing.

  ![image](https://github.com/Peter-Dumbari/Maths-Magician/assets/81468675/84fcf8d0-e1c7-407b-bb7b-3c5cb931547a)